src/core/include/openvino/op/roi_align.hpp

```diff
@@ -106,6 +106,16 @@ class OPENVINO_API ROIAlign : public Op {
     /// \param spatial_scale   Spatial scale factor used to translate ROI coordinates
     /// \param mode            Method of pooling - 'avg' or 'max'
     /// \param aligned_mode    Method of coordinates alignment - 'asymmetric', 'half_pixel_for_nn' or 'half_pixel'
+    ROIAlign(const Output<Node>& input,
```

Comment:
We already have a constructor using enums defined in this op and a way to conver from strings (EnumNames::as_enum), even in the client code. The string attributes can also be parsed using the visitor API (visit_attributes method).
Can you remove this constructor or is it necessary?

Reply:
Done